### ISSUE TITLE
Namespace breaks Index.submit

### ISSUE THREAD

**Issue by pwmcintyre** [splunk/splunk-sdk-java#101]
If you set a namespace in your session, you can't Index.submit.

I have a test framework which allows for test cases to submit data to Splunk, then ensure it's parsed correctly - but they share the same session.

```
// Create session
ServiceArgs loginArgs = new ServiceArgs();
loginArgs.setApp("testapp");
... set user/pass ...
service = Service.connect(loginArgs);

// Retrieve the index for the data
Index myIndex = service.getIndexes().get( "main" );

// Specify  values to apply to the event
Args eventArgs = new Args();
eventArgs.put("sourcetype", sourcetype);
eventArgs.put("host", "local");

// Submit an event over HTTP
myIndex.submit(eventArgs, data);
```

Error
```
com.splunk.HttpException: HTTP 404 -- Not Found
        at com.splunk.HttpException.create(HttpException.java:84)
        at com.splunk.HttpService.send(HttpService.java:452)
        at com.splunk.Service.send(Service.java:1293)
        at com.splunk.Receiver.submit(Receiver.java:169)
        at com.splunk.Index.submit(Index.java:1097)
```

Easy workaround is to have a seperate session for reading / writing. Maybe this is by design?

**Comment by shakeelmohamed** [comment]
Thanks for the bug report @pwmcintyre! The fix is pretty simple, we'd appreciate a pull request if you're interested in making one.

In `service.java` we need to change

```java
 protected String simpleReceiverEndPoint = "receivers/simple";
```

to 

```java
 protected String simpleReceiverEndPoint = "/services/receivers/simple";
```

See `String fullpath(String path, Args namespace)` in `service.java` for the URL path rules.

**Comment by pwmcintyre** [comment]
I will take a look when I can

**Comment by liketic** [comment]
@shakeelmohamed I sent a PR for this. Please review it.

**Comment by shakeelmohamed** [comment]
Merged PR to develop #118
